File: python3/2176.py
class Solution:
    def countPairs(self, nums: List[int], k: int) -> int:
        c = 0
        for i in range(len(nums)):
            for j in range(i):
                if nums[i] == nums[j] and (i * j) % k == 0:
                    c += 1
        return c

File: dart/2176.dart
class Solution {
    int countPairs(List<int> nums, int k) {
        int n = nums.length, c = 0;
        for (int i = 0; i < n; i++)
            for (int j = 0; j < i; j++)
                if (nums[i] == nums[j] && (i * j) % k == 0)
                    c++;
        return c;
    }
}
